src/main/kotlin/no/nav/medlemskap/regler/common/Personfakta.kt

```diff
@@ -253,7 +254,9 @@ class Personfakta(private val datagrunnlag: Datagrunnlag) {
     }
 
     fun medlemskapsPerioderOver12MndPeriodeDekning(): List<String> {
-        return medlemskapsPerioderOver12MndPeriode(true).map { it.dekning.orEmpty() }
+        return medlemskapsPerioderOver12MndPeriode(true).map {
+            dekningKoderCounter(it.dekning.orEmpty())
```

Comment:
Hva skjer når man sender inn tom streng? "Tåler" koden det? Kan vurdere egen default verdi for de tilfellene det er en tom streng/null slik at det blir bedre i Grafana.